third_party_licenses/README.md

```diff
@@ -0,0 +1,100 @@
+This folder contains a list of license file locations and types in `license_info.csv`  for all dependencies and transitive dependencies of the metadata docker image.
+
+The actual license contents of all dependencies shall be generated via the following command and put into the docker image:
+```
+$ python concatenate_license.py
```

Comment:
I'm curious why you decided to maintain the list and download them when building the image.
Are there any concerns committing concatenated license file into the repo?

I this approach because I worry about flakiness for image building. (Maybe it's not as bad as I thought, because github is stabler than other websites.)

Reply:
My concern is the correctness of the final license file.

The final license txt file is very large as I have seen it locally. I think mistakes of editing the file can happen if hundreds of lines are changed. If we check in the file, we will rely on the person do it right locally.

Having said that, flakiness of this script is also a concern if it happens quite often. I would leave this PR as it is now. If it makes the building process unstable, I can commit the final file and add a test to `diff` the script output with the final file.

Reply:
SG, thanks for explaining your reasoning behind